src/Kovan-DssSpell.t.sol

```diff
@@ -481,14 +694,24 @@ contract DssSpellTest is DSTest, DSMath {
             line:         5 * MILLION,
             dust:         100,
             pct:          200,
-            chop:         1300,
-            dunk:         500,
             mat:          17500,
-            beg:          300,
-            ttl:          1 hours,
-            tau:          1 hours,
-            liquidations: 1,
-            flipper_mom:  1
+            liqType:      "clip",
+            liqOn:        true,
+            chop:         1300,
+            cat_dunk:     0,
+            flip_beg:     0,
+            flip_ttl:     0,
+            flip_tau:     0,
+            flipper_mom:  0,
+            dog_hole:     10 * THOUSAND * RAD,
```

Comment:
Should probably change this to use natural units. IE drop the `RAD`.

Reply:
Done